js/config.js

```diff
@@ -12,6 +12,8 @@ defaultsDeep(systematic, {
     output_dir: 'dist',
     public_path: '/',
     enable_filename_hashing: true,
+    enable_babel: true,
+    es_libs: [],
```

Comment:
The project needs to declare its ES6 modules libs ?

Reply:
yep. Fortunately there's not many. It's either this or we can... take advantage of the "module" field in those dependencies package.json. I'll do this.